### PR TITLE
Added information about precompresssed files

### PR DESCRIPTION
**Pull request by eshanrnh**
## Description

Added information as mentioned in #[17615](https://github.com/umbraco/Umbraco-CMS/issues/17615)

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [x] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

CMS v15

## Deadline (if relevant)

Anytime

### CODE REVIEW

File: 15/umbraco-cms/fundamentals/setup/upgrading/version-specific/README.md [hunk 1]
```diff
@@ -54,6 +54,14 @@ You can now re-enable precompiled views and rebuild your site.
 
 If you have custom C# code that references the models this will also not build. You can either comment out your custom code temporarily until the models have been rebuilt or fix the models manually. To fix the models manually you need to find and replace `IPublishedSnapshotAccessor` with `IPublishedContentTypeCache`.
 
+**Handling Precompressed Files**
+
+When upgrading from Umbraco 14 to 15, you might notice that `JavaScript` and `CSS` files are automatically precompressed, resulting in additional `.br` and `.gz` files. This behavior is not specific to Umbraco but is introduced in ASP.NET Core version 9, where static files are fingerprinted and precompressed by default at build and publish time.
```
Comment (eshanrnh) [inline 15/umbraco-cms/fundamentals/setup/upgrading/version-specific/README.md:59]:
```suggestion
When upgrading from Umbraco 14 to 15, you might notice that `JavaScript` and `CSS` files are automatically precompressed, resulting in additional `.br` and `.gz` files. This behavior is not specific to Umbraco. It is introduced in ASP.NET Core version 9, where static files are fingerprinted and precompressed by default at build and publish time.
```

File: 15/umbraco-cms/fundamentals/setup/upgrading/version-specific/README.md [hunk 2]
```diff
@@ -54,6 +54,14 @@ You can now re-enable precompiled views and rebuild your site.
 
 If you have custom C# code that references the models this will also not build. You can either comment out your custom code temporarily until the models have been rebuilt or fix the models manually. To fix the models manually you need to find and replace `IPublishedSnapshotAccessor` with `IPublishedContentTypeCache`.
 
+**Handling Precompressed Files**
+
+When upgrading from Umbraco 14 to 15, you might notice that `JavaScript` and `CSS` files are automatically precompressed, resulting in additional `.br` and `.gz` files. This behavior is not specific to Umbraco. It is introduced in ASP.NET Core version 9, where static files are fingerprinted and precompressed by default at build and publish time.
+
+To disable this behavior, you can set `<CompressionEnabled>false</CompressionEnabled>` in your project file. If you are using Umbraco's templates - `dotnet new umbraco`, this setting is already applied.
```
Comment (eshanrnh) [inline 15/umbraco-cms/fundamentals/setup/upgrading/version-specific/README.md:61]:
```suggestion
To disable this feature, you can set `<CompressionEnabled>false</CompressionEnabled>` in your project file. If you are using Umbraco's templates - `dotnet new umbraco`, this setting is already included.
```

**Review by iOvergaard — APPROVED** [review]
Perfect.

File: 15/umbraco-cms/fundamentals/setup/upgrading/version-specific/README.md [hunk 3]
```diff
@@ -54,6 +54,16 @@ You can now re-enable precompiled views and rebuild your site.
 
 If you have custom C# code that references the models this will also not build. You can either comment out your custom code temporarily until the models have been rebuilt or fix the models manually. To fix the models manually you need to find and replace `IPublishedSnapshotAccessor` with `IPublishedContentTypeCache`.
 
+**Handling Precompressed Files**
+
+When upgrading from Umbraco 14 to 15, you might notice that `JavaScript` and `CSS` files are automatically precompressed, resulting in additional `.br` and `.gz` files. This behavior is not specific to Umbraco. It is introduced in ASP.NET Core version 9, where static files are fingerprinted and precompressed by default at build and publish time.
```
Comment (sofietoft) [inline 15/umbraco-cms/fundamentals/setup/upgrading/version-specific/README.md:59]:
```suggestion
When upgrading from Umbraco 14 to 15, you might notice that `JavaScript` and `CSS` files are automatically precompressed, adding additional `.br` and `.gz` files. This behavior is introduced in ASP.NET Core version 9, where static files are fingerprinted and precompressed by default at build and publish time.
```

**Pull request merged**